samples/winapi_drivelist/main.c

```diff
@@ -56,11 +58,25 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
-        Sleep(2000);
+    debugPrint("\nDone!");
+
+    goto unmount_e;
+
+unmount_e:
+    ret = nxUnmountDrive('E');
+    // If there was an error while unmounting
+    if (!ret) {
+        error = GetLastError();
+        debugPrint("\nCouldn't unmount E: drive! Error code: %x\n", error);
     }
 
+unmount_c:
+    ret = nxUnmountDrive('C');
+    if (!ret) {
+        error = GetLastError();
+        debugPrint("\nCouldn't unmount C: drive! Error code: %x\n. Trying to unmount E: drive...\n", error);
```

Comment:
Trying to unmount E: has nothing to do with unmount of C: having failed.
Therefore these should at least be separate messages (different `debugPrint` to differentiate them more clearly).

Also, the second message isn't even true, as @thrimbor pointed out, so it should be removed.